cromwellApiClient/src/main/scala/cromwell/api/model/CromwellStatus.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.api.model
```

Comment:
This was me I'm sure. I hate having these singleton things floating around but I also hate grab bags of fun which only ever grow in size. I was likely on the war path about the latter when I first put centaur together.